### ISSUE TITLE
Unexpectedly low quality transcription via HF Transformers (MMS)

### ISSUE THREAD

**Issue by andergisomon** [facebookresearch/fairseq#5190]
## Issue with unexpectedly low quality transcription

Suspiciously low quality transcription for ```lang='dtp'``` using MMS via Huggingface. Transcription is still bad even on CPU or CUDA.

### To Reproduce

1. Retrace the steps to run inference as in the MMS huggingface transformers documentation.
2. Load audio file as in code sample below.
3. Swap out the target language from English to ```dtp```.

#### Code sample

```
#load audio file
dtp_sample = Dataset.from_dict({"audio": ["/path/to/audio.wav"]}).cast_column("audio", Audio())
dtp_input = dtp_sample[0]["audio"]["array"]

processor.tokenizer.set_target_lang("dtp")
model.load_adapter("dtp")

inputs = processor(dtp_input, sampling_rate=16_000, return_tensors="pt").to("cuda")

with torch.no_grad():
    outputs = model(**inputs).logits

ids = torch.argmax(outputs, dim=-1)[0]
transcription = processor.decode(ids)
transcription

```

### Expected behavior

I have tested running the inference on CPU using [EasyMMS](https://github.com/abdeladim-s/easymms) and the transcription produced was reliably accurate, but the same cannot be reproduced via transformers.

### Environment

 - fairseq Version: main
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux): jupyter environment
 - How you installed fairseq (`pip`, source): pip
 - Python version: 3.9
 - CUDA/cuDNN version:
 - GPU models and configuration:

### Additional context

Everything was run in a Jupyter notebook. I have tested it both on Colab and Paperspace gradient but the issue remains.

**Comment by andergisomon** [comment]
Turns out I was just dumb. The model expects a sample rate of 16k.

Run the following command to change the sample rate:

```
ffmpeg -y -i /path/to/recording -b:a 16k -ar 16k /path/to/processed.wav
```
Then process it like below:

```
dtp_sample = Dataset.from_dict({"audio": ["/path/to/processed.wav"]}).cast_column("audio", Audio())
dtp_input = dtp_sample[0]["audio"]["array"]
```
